crates/bindings-csharp/Codegen/Codegen.csproj

```diff
@@ -29,6 +29,16 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- dev-dependency to add internal C# classes and attributes not included in .NET Standard -->
+    <PackageReference Include="PolySharp" Version="1.14.1" PrivateAssets="all" />
```

Comment:
Is this needed inside codegen? It should normally run on a recent enough version of dotnet no?